src/components/QueryFileUploader/utils/getFileInfo.js

```diff
@@ -0,0 +1,11 @@
+import { getFileExtension } from '@folio/stripes-data-transfer-components';
+import { SUPPORTED_FILE_EXTENSIONS } from '../../../utils/constants';
+
+export const getFileInfo = file => {
+  const fileType = getFileExtension(file).substring(1);
+
+  return {
+    isTypeSupported: !!SUPPORTED_FILE_EXTENSIONS[fileType.toUpperCase()],
```

Comment:
```suggestion
    isTypeSupported: Boolean(SUPPORTED_FILE_EXTENSIONS[fileType.toUpperCase()]),
```
Your solution looks good but the Boolean(value) is used in our code base. Let's use it for consistency.